velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +495,27 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
+struct WilsonIntervalUpperFunction {
+  FOLLY_ALWAYS_INLINE void call(
+      double& result,
+      const int64_t& successes,
+      const int64_t& trials,
+      const double& z) {
+    result = wilsonInterval<true /*upper*/>(successes, trials, z);
+  }
+};
+
+template <typename T>
+struct WilsonIntervalLowerFunction {
+  FOLLY_ALWAYS_INLINE void call(
+      double& result,
+      const int64_t& successes,
+      const int64_t& trials,
+      const double& z) {
+    result = wilsonInterval<false /*upper*/>(successes, trials, z);
```

Comment:
typo in comment - should say `lower`.

Reply:
The intention here was to match with the convention used in other places in the velox code, for example in StringFunctions.cpp [line 436](https://github.com/facebookincubator/velox/blob/cae11277f880a372d0a18df05f46180febb63ee9/velox/functions/prestosql/StringFunctions.cpp#L436) and [line 441](https://github.com/facebookincubator/velox/blob/cae11277f880a372d0a18df05f46180febb63ee9/velox/functions/prestosql/StringFunctions.cpp#L441). The comment next to the template boolean is `/*isLower*/` for both `true` and `false` values. Similarly in StringWriter.h [line 30](https://github.com/facebookincubator/velox/blob/cae11277f880a372d0a18df05f46180febb63ee9/velox/expression/StringWriter.h#L30) and [line 164](https://github.com/facebookincubator/velox/blob/cae11277f880a372d0a18df05f46180febb63ee9/velox/expression/StringWriter.h#L164), the comment is `/*reuseInput*/` in both cases.

Perhaps I should make it more clear that the template variable is being referenced. I'll change the name to `isUpper`. Is this okay?

Reply:
Yes. I misunderstood what the comment meant (I guess borne by the confusing name). I thought it indicated which template (upper or lower) based on the boolean value is executed and not the actual name of the template parameter.